.github/workflows/pr-check-docs.yml

```diff
@@ -21,9 +22,27 @@ jobs:
       with:
         node-version: 12
 
-    - name: Run npm commands
+    - name: Build docs
       if: github.repository == 'ExpediaGroup/graphql-kotlin'
       run: |
         cd website
         npm install
         npm run build
+
+  check-perf-tests:
```

Comment:
why would we want to run perf tests if someone updates docs?

Reply:
maybe we should split those up?

Reply:
We may just be able to delete the perf tests here that use npm. Instead we may want some other client. This was an add in to debug our subscriptions.